Integrations/python/deephaven/transferrer/__init__.py

```diff
@@ -61,21 +61,29 @@ def table_to_numpy_2d(idx, cols, np_dtype = None):
         np_dtype = np.intc
 
     if np_dtype == np.bool_:
```

Comment:
I would reorder this section to be ordered by type (e.g. int) from small to large (e.g. float, double).  It just makes reading the code easier later.